python/lsst/ts/standardscripts/base_track_target.py

```diff
@@ -278,14 +299,22 @@ async def configure(self, config):
 
         if hasattr(self.config, "slew_icrs"):
             self.slew_type = SlewType.ICRS
+        elif hasattr(self.config, "slew_planet"):
+            self.slew_type = SlewType.PLANET
         elif hasattr(self.config, "find_target"):
             self.slew_type = SlewType.AZEL
 
         self.log.debug(f"Slew type: {self.slew_type!r}.")
 
         await self.configure_tcs()
 
-        self.config.rot_type = getattr(RotType, self.config.rot_type)
+        if self.slew_type == SlewType.PLANET and self.config.rot_type != "Sky":
+            self.log.warning(
+                f"Rotator type must be Sky for slew_type={self.slew_type}. "
+            )
+        else:
+            self.config.rot_type = getattr(RotType, self.config.rot_type)
```

Comment:
I think it would be better if you did this before the `if` statement, then, compare `self.config.rot_type` with `RotType.Sky`. Also, since you are not raising an exception, but issuing a warning message, maybe rephrase the message a bit.

Consider the following:

```
self.config.rot_type = getattr(RotType, self.config.rot_type)

if self.slew_type == SlewType.PLANET and self.config.rot_type != RotType.Sky:
    self.log.warning(
        f"Slew_type={self.slew_type!r} only works for {RotType.Sky!r}, "
        f"got {self.config.rot_type!r}. Ignoring."
    )
```